light_curves/multiband_lc.ipynb

```diff
@@ -564,13 +560,13 @@
    "outputs": [],
    "source": [
     "# # Save or load the data frame. This is mainly for debugging.\n",
-    "# if SAVEDF:\n",
-    "#     df_lc.pickle(pickle_file_name)\n",
-    "#     print(\"Pickle file saved!\")\n",
-    "# else:\n",
-    "#     df_lc = MultiIndexDFObject()\n",
-    "#     df_lc.load_pickle(pickle_file_name)\n",
-    "#     print(\"Pickle file loaded!\")"
+    "if SAVEDF:\n",
+    "    df_lc.pickle(pickle_file_name)\n",
+    "    print(\"Pickle file saved!\")\n",
+    "else:\n",
+    "    df_lc = MultiIndexDFObject()\n",
+    "    df_lc.load_pickle(pickle_file_name)\n",
+    "    print(\"Pickle file loaded!\")"
```

Comment:
This cell errors for me with 

```
ModuleNotFoundError: No module named 'pandas.core.indexes.numeric'
```

and it also overwrites `df_lc` with an empty dataframe so that it's no longer usable for plotting.

Since this is really just for debugging, I suggest that be commented out.

Reply:
I remove that cell in my cleanup PR, so no need to worry about it here.

Reply:
yes, sorry, I forgot to comment that out again. It is not commented out.
Since that was a minor fix, I'm going to merge now.